lib/td/command/runner.rb

```diff
@@ -94,6 +96,12 @@ def run(argv=ARGV)
       endpoint = e
     }
 
+    op.on('-E', '--import-endpoint API_IMPORT_SERVER', "specify the URL for API Import server to use (default: https://api-import.treasuredata.com).") { |e|
```

Comment:
Could you avoid `-E`?

Reply:
Do you mean to use different character? (for example, `-n`?)

Reply:
Pushed a commit to use `-i` for "import".
@nurse How do you feel on it?

Reply:
I mean just remove one character option.

Reply:
Pushed the commit to remove it.